common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -119,7 +121,7 @@ public static void setEnabled(boolean enabled) {
      * Returns {@code true} if resource leak detection is enabled.
      */
     public static boolean isEnabled() {
-        return getLevel().ordinal() > Level.DISABLED.ordinal();
```

Comment:
@ninja- - what @normanmaurer said. However if we make the other changes @normanmaurer recommended (using `ResourceLeakDetector.isEnabled()` to avoid calling `touch`) it is possible we will still get some improvement....but lets do a quick benchmark just to be sure.